### PR TITLE
Contributing.md

### LINKED ISSUE #39: Add CONTRIBUTING.md
Add a guide for people interested in contributing to the project, especially those who did not attend earlier HackNights.

### PR DESCRIPTION
**Pull request by melissachen2000**
Resolves #39. Adds a basic contributing.md with information about how to get the project on your local machine, how to run the project, the process for proposing new features or reporting bugs, and how to contribute to the projects via PRs.

### CODE REVIEW

**Review by brianstrauch — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Contributing
+
+When contributing to this repository, please keep in mind the following information.
+
+We highly encourage you to join our [Slack channel on the UIUC OSS Slack](https://illinois-oss.slack.com) to discuss features, seek development support, and be alerted about our bi-weekly dev meetups. 
+
+## To develop on your machine
+
+1. Make a fork of the repository by clickin the fork button at the top right of the page
```
Comment (sathreddy) [inline CONTRIBUTING.md]:
```suggestion
1. Make a fork of the repository by _clickin_ the fork button at the top right of the page
```
Me likes 🤪

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+2. Clone the forked repository onto your local machine by running `git clone https://github.com/[YOUR GITHUB USERNAME]/clickin`
+3. If you have not already, install [Node.js](https://nodejs.org/en/download/) and [Python 3](https://www.python.org/downloads/)
+4. If developing frontend, from the repository root directory, run `npm install`
+5. If developing backend, from the repository root directory, run `pip install -r server/requirements.txt`
+6. Comment on any open issue to claim it, and begin developing!
+
+## To run on your machine
+1. Run the server using `python3 server.py`
```
Comment (sathreddy) [inline CONTRIBUTING.md]:
```suggestion
1. Run the backend using `python3 server.py` in the `server/` directory
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+2. Run the webapp in your browser using `npm start`
```
Comment (sathreddy) [inline CONTRIBUTING.md]:
```suggestion
2. Run the webapp in your browser using `npm start` in the `app/` directory
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (melissachen2000) [reply CONTRIBUTING.md]: Typo :(

**Review by sathreddy — COMMENTED** [review]
Lmk what you think of the suggestions.

File: CONTRIBUTING.md (hunk 1 above)
Comment (sathreddy) [reply CONTRIBUTING.md]: Let's keep it! <3

Comment (melissachen2000) [reply CONTRIBUTING.md]: I just fixed the typo....but I'll change it back.

**Comment by melissachen2000** [comment]
Made the suggested changes!

**Review by sathreddy — APPROVED** [review]

**Pull request merged**